mentors/README.md: Add comment about whitespace (#912)
exercism/website-copy: mentors/README.md

## Add your bio
Choose the JSON file that starts with the first character of your GitHub username.

> Use the following format. 

Reviewer: ```suggestion
Use the following format.
```
Super minor, not sure how the space got inserted, but lets eliminate it so only what you intend to change ends up changed.

Author: Oh, probably my fault... Thanks for spotting this one.

      "bio": "I love Go for its simplicity and consistency. I've written Go professionally, and also use it for open source projects."
    }

> Note: Watch out for the amount of needed whitespace, after the { every line is indented by 2 spaces until the }.

Reviewer: ```suggestion
Note: After the opening `{`, every line is indented by 2 spaces until the closing `}`.
```
I wonder if it would be worth while to add to the example JSON above, to include the square brackets.  They are not to be indented.  I think with them omitted from the example above it adds to the confusion regarding indentations.

Reviewer: Indentation should be progressive.  The lines between the opening `{` and closing `}` should be two spaces further in, 4 total.

Reviewer: As an aside this will become moot at some point as this process is slated to be folded into the site directly.  Making it unnecessary for mentors to submit PRs containing bio information (I almost said matter!).  :smile:

Author: Thanks for the edit. Interesting, yes submitting it on the site will probably be a bit easier :)
